tools/check_jsonnet

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -e
+
+success=1
+JSONNET_FILES=$(git ls-files data/ | grep -E '.json\?net$i|.libson\?net$' | xargs echo)
```

Comment:
I don't think it is correct the reject, does it really match `.json` ? the `\?` only applies to the the `n` right?

Reply:
This is the regex copied from existed check file, for our test, we can check distinguish extension there.